src/NServiceBus.NHibernate/Outbox/OutboxCleaner.cs

```diff
@@ -36,6 +37,25 @@ protected override Task OnStart(IMessageSession busSession)
                 throw new Exception("Invalid value in \"NServiceBus/Outbox/NHibernate/FrequencyToRunDeduplicationDataCleanup\" AppSetting. Please ensure it is a TimeSpan.");
             }
 
+            var key = "NServiceBus/Outbox/NHibernate/TimeToWaitBeforeTriggeringCriticalErrorWhenCleanupTaskFails";
+
+            configValue = ConfigurationManager.AppSettings.Get(key);
+
+            if (configValue == null)
+            {
+                timeToWaitBeforeTriggeringCriticalError = TimeSpan.FromMinutes(2);
```

Comment:
Extract `TimeSpan.FromMinutes(2)` as a readonly field to give it more visibility?